### PR TITLE
Artik resin uboot

### PR DESCRIPTION
**Pull request by agherzan**

### CODE REVIEW

File: layers/meta-resin-artik/recipes-bsp/u-boot/patches/artik-integrate-with-resin-configuration.patch [hunk 1]
```diff
@@ -0,0 +1,35 @@
+From b04f97cf90344af54a620d2de7587760853206fe Mon Sep 17 00:00:00 2001
+From: Andrei Gherzan <andrei@resin.io>
+Date: Wed, 4 Jan 2017 16:09:39 +0100
+Subject: artik_common.h: Integrate with resin environment configuration
+
+Artik (5 and 10) machine specific integration of resin environment
+configuration.
+
+Signed-off-by: Andrei Gherzan <andrei@resin.io>
```
Comment (michal-mazurek) [inline layers/meta-resin-artik/recipes-bsp/u-boot/patches/artik-integrate-with-resin-configuration.patch:9]: Upstream-status unneeded here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Add resinhup machine specific support in uboot [Andrei]
```
Comment (michal-mazurek) [inline CHANGELOG.md:7]: I think that U-Boot must be written capitalized and with a dash

**Review by petrosagg — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (agherzan) [reply CHANGELOG.md:7]: Fixed.

File: layers/meta-resin-artik/recipes-bsp/u-boot/patches/artik-integrate-with-resin-configuration.patch (hunk 1 above)
Comment (agherzan) [reply layers/meta-resin-artik/recipes-bsp/u-boot/patches/artik-integrate-with-resin-configuration.patch:9]: Fixed.

**Review by michal-mazurek — APPROVED** [review]

**Review by floion — APPROVED** [review]

**Comment by agherzan** [comment]
Implements https://github.com/resin-io/hq/pull/521 on artik

**Pull request merged**